docs/useMediatedState.md

```diff
@@ -0,0 +1,42 @@
+# `useMediatedState`
+
+A lot like the standard `useState`, but with mediation process.
+
+## Usage
+```ts
+import * as React from 'react';
+import { useMediatedState } from '../useMediatedState';
+
+const InputMediator = s => s.replace(/[\s]+/, ' ');
```

Comment:
Can you update the regex and rename `InputMediator` to `inputMediator`?

Reply:
I've updated %) strange that you've accessed the old one.
About function name i'll forcepush now.

Reply:
This comment is in the mardown file of your hook, not the story :)

Reply:
\*facepalm.jpg*